adobe-apiplatform/user-sync.py
# Moving CLI config to API Platform

## Summary
* Moving CLI config to API Platform. (UST examples CLI to produce full set)


## Testing Steps
* Run make standalone command
* The config files should be generated in appropriate folders under the resources/examples
* Run init method with generate all command line arguments
* The config files should be generated in appropriate folders in the cwd



Fixes #193
